Tasks/AzureFileCopy/task.loc.json

```diff
@@ -87,24 +87,21 @@
       "visibleRule": "Destination = AzureVMs"
     },
     {
-      "name": "ResourceFilteringMethod",
-      "type": "radio",
-      "label": "ms-resource:loc.input.label.ResourceFilteringMethod",
-      "required": false,
-      "defaultValue": "machineNames",
-      "options": {
-        "machineNames": "Machine Names",
-        "tags": "Tags"
-      },
+      "name": "vmsAdminUserName",
+      "type": "string",
+      "label": "ms-resource:loc.input.label.vmsAdminUserName",
+      "defaultValue": "",
+      "required": true,
+      "helpMarkDown": "ms-resource:loc.input.help.vmsAdminUserName",
       "visibleRule": "Destination = AzureVMs"
     },
     {
-      "name": "MachineNames",
+      "name": "vmsAdminPassword",
```

Comment:
we have to modify existing rm onprem tests for them as they donot have vmusername n password